bin/dcps_tests.lst

```diff
@@ -294,6 +294,22 @@ tests/DCPS/Thrasher/run_test.pl low rtps: !DCPS_MIN RTPS !LYNXOS
 tests/DCPS/Thrasher/run_test.pl medium rtps: !DCPS_MIN RTPS !LYNXOS
 tests/DCPS/Thrasher/run_test.pl high rtps: !DCPS_MIN RTPS !LYNXOS
 tests/DCPS/Thrasher/run_test.pl aggressive rtps: !DCPS_MIN RTPS !LYNXOS
+tests/DCPS/Thrasher/run_test.pl single durable: !DCPS_MIN !LYNXOS
+tests/DCPS/Thrasher/run_test.pl double durable: !DCPS_MIN !LYNXOS
+tests/DCPS/Thrasher/run_test.pl triangle durable: !DCPS_MIN !LYNXOS
+tests/DCPS/Thrasher/run_test.pl default durable: !DCPS_MIN !LYNXOS
+tests/DCPS/Thrasher/run_test.pl low durable: !DCPS_MIN !LYNXOS
+tests/DCPS/Thrasher/run_test.pl medium durable: !DCPS_MIN !LYNXOS
+tests/DCPS/Thrasher/run_test.pl high durable: !DCPS_MIN !LYNXOS
+tests/DCPS/Thrasher/run_test.pl aggressive durable: !DCPS_MIN !LYNXOS
```

Comment:
The equivalent lines on master have changed a bit since you made this PR.  The tests that are not `rtps` have an additional ` !OPENDDS_SAFETY_PROFILE` here.